packages/paste-core/components/checkbox/__tests__/checkbox.test.tsx

```diff
@@ -48,12 +51,15 @@ describe('Checkbox', () => {
   });
 
   it('should render as checked', () => {
-    const {getByLabelText} = render(
+    const {getByLabelText, container} = render(
       <Checkbox {...defaultProps} checked onChange={NOOP}>
         foo
       </Checkbox>
     );
     expect((getByLabelText('foo') as HTMLInputElement).checked).toBeTruthy();
+
+    const checkIcon = container.querySelector('[data-paste-element="CHECKBOX_ICON"');
```

Comment:
Does the checkbox use a React Portal or something else that causes the target element to render outside of the main render return target?

Reply:
Nope, I just followed the format of how the customization tests target elements.